components/ItemGrid/LoadVideoContentTask.bs

```diff
@@ -169,9 +181,10 @@ sub LoadItems_AddVideoContent(video as object, mediaSourceId as dynamic, audio_s
         end if
     end if
 
-    logoImageExists = api.items.HeadImageURLByName(logoLookupID, "logo")
+    logoSource = videotype = "tvchannel" ? "primary" : "logo" ' TV Channel logos are stored under '/primary' instead of '/logo'
```

Comment:
Import ImageType enum
```
import "pkg:/source/enums/ImageType.bs"
```

```suggestion
    logoSource = isStringEqual(videotype, ItemType.TVCHANNEL) ? ImageType.PRIMARY : ImageType.LOGO ' TV Channel logos are stored under '/primary' instead of '/logo'
```